rfc/0084-app-service-level-telemetry.md

```diff
@@ -0,0 +1,516 @@
+      <td>&lt;=2.5s<br><code style="white-space:nowrap">le="2.5"</code></td>
+      <td>&gt;2.5s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+    <tr>
+      <td>sdk_kv_mutation_nondurable</td>
+      <td>&lt;=1ms<br><code style="white-space:nowrap">le="0.001"</code></td>
+      <td>&lt;=10ms<br><code style="white-space:nowrap">le="0.01"</code></td>
+      <td>&lt;=100ms<br><code style="white-space:nowrap">le="0.1"</code></td>
+      <td>&lt;=500ms<br><code style="white-space:nowrap">le="0.5"</code></td>
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1"</code></td>
+      <td>&lt;=2.5s<br><code style="white-space:nowrap">le="2.5"</code></td>
+      <td>&gt;2.5s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+    <tr>
+      <td>sdk_kv_mutation_durable</td>
+      <td>&lt;=10ms<br><code style="white-space:nowrap">le="0.01"</code></td>
+      <td>&lt;=100ms<br><code style="white-space:nowrap">le="0.1"</code></td>
+      <td>&lt;=500ms<br><code style="white-space:nowrap">le="0.5"</code></td>
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1"</code></td>
+      <td>&lt;=2s<br><code style="white-space:nowrap">le="2"</code></td>
+      <td>&lt;=10s<br><code style="white-space:nowrap">le="5"</code></td>
+      <td>&gt;10s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+    <tr>
+      <td>sdk_query</td>
+      <td></td>
+      <td>&lt;=100ms<br><code style="white-space:nowrap">le="0.1"</code></td>
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1"</code></td>
+      <td>&lt;=10s<br><code style="white-space:nowrap">le="10"</code></td>
+      <td>&lt;=30s<br><code style="white-space:nowrap">le="30"</code></td>
+      <td>&lt;=75s<br><code style="white-space:nowrap">le="75"</code></td>
+      <td>&gt;75s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+    <tr>
+      <td>sdk_search</td>
+      <td></td>
+      <td>&lt;=100ms<br><code style="white-space:nowrap">le="0.1"</code></td>
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1"</code></td>
+      <td>&lt;=10s<br><code style="white-space:nowrap">le="10"</code></td>
+      <td>&lt;=30s<br><code style="white-space:nowrap">le="30"</code></td>
+      <td>&lt;=75s<br><code style="white-space:nowrap">le="75"</code></td>
+      <td>&gt;75s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+    <tr>
+      <td>sdk_analytics</td>
+      <td></td>
+      <td>&lt;=100ms<br><code style="white-space:nowrap">le="0.1"</code></td>
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1"</code></td>
+      <td>&lt;=10s<br><code style="white-space:nowrap">le="10"</code></td>
+      <td>&lt;=30s<br><code style="white-space:nowrap">le="30"</code></td>
+      <td>&lt;=75s<br><code style="white-space:nowrap">le="75"</code></td>
+      <td>&gt;75s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+  </tbody>
+</table>
+
```

Comment:
Are the thresholds for eventing and management the same as query/search/analytics?

Reply:
Yes, I will update those for eventing.